efopen/ef_generate.py

```diff
@@ -344,6 +344,34 @@ def conditionally_create_profile(role_name, service_type):
   else:
     print_if_verbose("instance profile already contains role: {}".format(role_name))
 
+
+def get_managed_policy_arn(policy_name):
+  """
+  Retrieves the ARN of an AWS managed policy specified in the aws_managed_policies field of the service_registry.
+  Args:
+    policy_name: name of the policy whose ARN we want to retrieve
+  Raises:
+    NameError: no ARN could be found for the specified policy name
+  """
+  try:
+    arn = 'arn:aws:iam::aws:policy/{}'.format(policy_name)
+    CLIENTS["iam"].get_policy(PolicyArn='arn:aws:iam::aws:policy/{}'.format(policy_name))
```

Comment:
Why not using `PolicyArn=arn` ?

```suggestion
    CLIENTS["iam"].get_policy(PolicyArn=arn)
```

Reply:
Thanks for catching that